bedrock/legal/templates/legal/index.html

```diff
@@ -81,8 +81,16 @@ <h1 id="main-feature" class="center large">{{ self.page_title() }}</h1>
 
 {% block email_form %}
   {% if LANG.startswith('en-') %}
```

Comment:
Suggest moving the `{% if LANG.startswith('en-') %}` conditional inside the `<aside id="newsletter-subscribe">` container to avoid repeating the markup?